drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/rpu_fw_patches.h

```diff
@@ -1,804 +1,970 @@
 const unsigned char wifi_nrf_lmac_patch_pri_bimg[] = {
-	0xba, 0xda, 0xba, 0xab, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00, 0x00, 0x00,
-	0xc0, 0xcc, 0x00, 0x80, 0x02, 0x00, 0x00, 0x00, 0xff, 0xff, 0x00, 0x00,
-	0x7d, 0xea, 0x00, 0x00, 0x02, 0xd4, 0x30, 0x32, 0x5c, 0x15, 0x08, 0x80,
-	0x01, 0x00, 0xa9, 0xbe, 0x02, 0xd4, 0x18, 0x30, 0x42, 0x14, 0x08, 0x80,
-	0x01, 0x00, 0x8e, 0x8b, 0x02, 0xd4, 0xb0, 0x31, 0x76, 0x10, 0x08, 0x80,
-	0x01, 0x00, 0xf7, 0x0b, 0x02, 0xd4, 0x08, 0x30, 0x1e, 0x17, 0x08, 0x80,
-	0x01, 0x00, 0x9f, 0x23, 0x02, 0xd4, 0x28, 0x30, 0x18, 0x14, 0x08, 0x80,
-	0x01, 0x00, 0xc4, 0x3b, 0x02, 0xd4, 0x52, 0x30, 0xee, 0x13, 0x08, 0x80,
-	0x01, 0x00, 0x3d, 0x61, 0x02, 0xd4, 0xfc, 0x30, 0x5c, 0x12, 0x08, 0x80,
-	0x01, 0x00, 0x78, 0xa3, 0x00, 0x00, 0x00, 0x00, 0xff, 0xff, 0xff, 0xff,
-	0x03, 0x00, 0xba, 0x3e
+		0xba, 0xda, 0xba, 0xab, 0x00, 0x00, 0x00, 0x10, 0x00, 0x00, 0x00, 0x00,
```

Comment:
nit: Is the extra indent needed here? One tab should be enough. Here and in other places.